pyproject.toml

```diff
@@ -105,8 +105,8 @@ where = ["src"]
 
 # https://pypi.org/project/bumpver
 [tool.bumpver]
-current_version = "0.1.16"
-version_pattern = "MAJOR.MINOR.PATCH[PYTAGNUM]"
+current_version = "1.0.0rc0"
+version_pattern = "MAJOR.MINOR.PATCH[.PYTAGNUM]"
```

Comment:
I don't think we want the `.` added back in; iirc, PyPI only recognizes RC versions without the `.`